setup.cfg

```diff
@@ -9,10 +9,10 @@ tag_build =
 universal=1
 
 [bdist_rpm]
-release = 1
+release = 2
```

Comment:
Can you please leave this value at 1? Or is this currently important for your build. If so its ok an we will reset it to 1 in the next release.